main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -581,12 +576,26 @@ object JvmOps {
           case (sacc, e) => sacc ++ visitExp(e)
         }
 
-      case Expression.NewObject(_, _, _, methods, _) =>
-        methods.foldLeft(Set.empty[ClosureInfo]) {
-          case (sacc, JvmMethod(_, _, clo, _, _)) => sacc ++ visitExp(clo)
-        }
+      case Expression.Intrinsic0(op, _, _) => op match {
+
+        case IntrinsicOperator0.Cst(_) => Set.empty
+
+        case IntrinsicOperator0.Region => Set.empty
+
+        case IntrinsicOperator0.RecordEmpty => Set.empty
+
+        case IntrinsicOperator0.GetStaticField(_) => Set.empty
+
+        case IntrinsicOperator0.NewObject(_, _, methods) =>
```

Comment:
I take it that this is necessary for the code to compile?

Reply:
(which kind of breaks the idea that we should not have to match in each intrinsic)

Reply:
I might be able to pre-compute `ClosureInfo` thus solving this.

Reply:
I was able to precompute closure info